ext/src/ruby_api/wasi_ctx.rs

```diff
@@ -29,96 +25,6 @@ pub struct WasiCtx {
 type RbSelf = Obj<WasiCtx>;
 
 impl WasiCtx {
-    /// @yard
-    /// Create a new deterministic {WasiCtx}. See https://github.com/Shopify/deterministic-wasi-ctx for more details
-    /// @return [WasiCtx]
-    pub fn deterministic() -> Self {
-        Self {
-            inner: RefCell::new(wasi_deterministic_ctx()),
```

Comment:
There is no equivalent for this call with `wasmtime-wasi`. Instead there's a method that accepts a `WasiCtxBuilder` as input.